README.md

```diff
@@ -1,10 +1,46 @@
 # oranda
-> beautiful custom websites for your CLI
+> 🎁 generate beautiful landing pages for your developer tools
 
-## Usage
+[![crates.io](https://img.shields.io/crates/v/oranda.svg)](https://crates.io/crates/oranda)
+![CI](https://github.com/axodotdev/oranda/actions/workflows/ci/badge.svg?branch=main)
+![release](https://github.com/axodotdev/oranda/actions/workflows/release/badge.svg?branch=main)
+![docs](https://github.com/axodotdev/oranda/workflows/actions/docs/badge.svg?branch=main)
 
-- `build`: generates a static site based on project and/or custom configuration.
-  optionally takes a `--path` flag, otherwise looks in the current directory
-- `serve`: starts a local server to preview site generated from `build` command.
-  uses `dist_dir` to find assets, which defaults to `public`. optionally takes
-  a `--port` flag, otherwise uses `7979`.
+`oranda` is a strongly opinionated static-site generator that is designed for
+developers who are self-publishing tools and would like a website but don't
+want to build one from scratch.
+
+`oranda` uses `oranda` so you can checkout a live example [here]!
+
+## Installation
+
+To install `oranda`, please visit the `oranda` [website]- which is generated by
+`oranda`!
+
+## Quickstart
+
+```sh
+# build your site
+> oranda build
+
+# start a server to checkout a local version of your built site in a browser
+> oranda serve
+```
+
+## Configuration
+
+If you'd like to configure `oranda`, place an `oranda.json` file in the root of
+your project and fill it with the configuration you'd like. Check out the [docs]
+to learn more about your configuration options!
+
+[docs]: TBD
+
+## Installers: integrating with `cargo-dist`
+
+`oranda` is built to work alongside `cargo-dist` which is a tool that builds
+distributable artifacts for your Rust applications. To tell `oranda` you are
+using `cargo-dist` you can add this to your `oranda.json`:
+
+```json
+"cargo-dist": true
```

Comment:
It's 

```
artifacts: {
 cargo-dist: true
}
```

Reply:
haha thanks! i just put in placeholder, but it's good to know the real config :D